config_files/training/config_lorem_ipsum.yaml

```diff
@@ -178,7 +178,25 @@ wrapped_model:
     sharding_strategy: FULL_SHARD
     block_names: [GPT2Block]
 
-model:
+model: 
+  component_key: model
+  variant_key: weight_initialized
+  config:
+    model:
+      instance_key: model_raw
+      pass_type: BY_REFERENCE
+    weight_initializer:
+      component_key: weight_initialization
+      variant_key: composed
+      config:
+        model_type: gpt2
+        weight_init_type: scaled_embed
+        mean: 0.0
+        std: 0.02
+        hidden_dim: ${model_raw.config.n_embd}
```

Comment:
Since `std != 'auto'`, the `hidden_dim` is not used and should not be specified. See also other comment.

Reply:
added pydantic checks for this case:
https://github.com/Modalities/modalities/commit/eea99bc1ba86923c8e9fbfa410c8bc502e8598f4